hawkbit-repository/hawkbit-repository-api/src/main/java/org/eclipse/hawkbit/repository/ControllerManagement.java

```diff
@@ -135,6 +135,24 @@
     @PreAuthorize(SpringEvalExpressions.IS_CONTROLLER)
     Action addUpdateActionStatus(@NotNull @Valid ActionStatusCreate create);
 
+    /**
+     * Retrieves all active {@link Action}s of a specific target.
+     * 
+     * @param pageable
+     *            the page request parameter for paging and sorting the result
+     * @param controllerId
+     *            the target associated with the actions
```

Comment:
Target identified by it's `{@linkplain Target#getControllerId controllerId}`...

Reply:
{@linkplain Target#getControllerId controllerId} is not working. Not sure if it because the Target is an interface.